src/main/java/net/fabricmc/stitch/commands/tinyv2/CommandMergeTinyV2.java

```diff
@@ -87,7 +87,7 @@ private void merge(TinyFile inputA, TinyFile inputB) {
             String sharedName = tinyClass.getClassNames().get(0);
             TinyClass matchingClass = inputBClassesByFirstNamespaceName.get(sharedName);
 
-            if (matchingClass == null || matchingClass.getClassNames().get(1).equals("")) {
+            if (matchingClass == null || matchingClass.getClassNames().get(1).isEmpty()) {
```

Comment:
Just check if two class names equal. Checking empty doesn't work!

Reply:
it does work